package.json

```diff
@@ -2,13 +2,18 @@
   "name": "focus-trap",
   "version": "2.4.3",
   "description": "Trap focus within a DOM node.",
-  "main": "index.js",
+  "main": "dist/focustrap.min.js",
   "types": "index.d.ts",
   "scripts": {
     "lint": "eslint .",
     "demo-bundle": "browserify demo/js/index.js -o demo/demo-bundle.js",
+    "clean": "del-cli !dist/.gitkeep dist/*",
+    "build-dev": "browserify index.js -s focusTrap > dist/focustrap.js",
+    "minify": "uglifyjs dist/focustrap.js > dist/focustrap.min.js",
+    "build": "npm run build-dev && npm run minify",
     "start": "budo demo/js/index.js:demo-bundle.js --dir demo --live",
-    "test": "npm run lint"
+    "test": "npm run lint",
+    "prepare": "npm run build"
```

Comment:
How about `prepublishOnly`?

Reply:
Yeah, I took wrong turn here, I'll change ti to `prepublishOnly`. :+1: